objects/src/item/Placement.java

```diff
@@ -126,32 +71,40 @@ private void createNullList(ArrayList<Student> onePriorityList, ArrayList<Studen
             }
         }
     }
-
     private void placePriorityLists(ArrayList<Student> priorityList, ArrayList<Options> optlist) {
         for(Student stu:priorityList){
-            int checking = 0;
-            for( int i = 0;i<stu.getStudentChoices().size();i++){
-                if(checking == 0){
-                    for(Options opt:optlist){
-                        if (opt.getCourseName().equals(stu.getStudentChoices().get(i)) && stu.getStatus().equals("") && opt.getEmptySeats()!= 0) {
-                            opt.addStudentToList(stu);
-                            stu.setAssignedOption(opt.getCourseName());
-                            checking++;
-                            break;
+            if(stu.getStudentChoices().size()>0){
```

Comment:
Did you read the else statement? I put it this way for that reason. If student it rejected because of status, I want to know why. If I added it with &&, then I wouldn't be able to confirm why they were rejected

Reply:
I did have it that way before, but I'm just preparing for all possibilities at this point. The code will get even longer and more jumbly

Reply:
Oh i see. Thanks for clarifying that.

Reply:
I just have a bad feeling itll come bite us later. But we can deal with it then :). Good job on the code 👍